htslib/1.16/README.md

```diff
@@ -0,0 +1,23 @@
+# htslib container
+
+Main tool:
+
+* [https://www.htslib.org/](https://www.htslib.org/)
+* [GitHub](https://github.com/samtools/htslib)
+
+Additional tools:
+
+* perl 5.30.0
+
+## Example Usage
+
+```bash
+# determine file formats for various BAM and SAM files
+$ htsfile tests/SARS-CoV-2/SRR13957123.primertrim.sorted.bam 
+tests/SARS-CoV-2/SRR13957123.primertrim.sorted.bam:     BAM version 1 compressed sequence data
+
+$ htsfile ce_tag_padded.sam
+ce_tag_padded.sam:      SAM version 1.4 sequence text

```

Comment:
I just now realized I forgot to include a readme for 1.15. Could you add

```
# compresses sample.fastq to sample.fastq.gz
$ bgzip sample.fastq
```

to the htslit/1.16/README.md? It's what I mainly use this image for.

Reply:
Sure!